js/cm-api-v2/PatchSnippet.js

```diff
@@ -0,0 +1,54 @@
+// DocSection: cm_api_v2_patch_snippet
+// Tip: Find more about JS/TS SDKs at https://docs.kontent.ai/javascript
+// Using ES6 syntax
+import { ManagementClient } from '@kentico/kontent-management';
+
+const client = new ManagementClient({
+  projectId: '<YOUR_PROJECT_ID>',
+  apiKey: '<YOUR_API_KEY>'
+});
+
+client.modifyContentTypeSnippet()
```

Comment:
The params of this method are indented in a different style compared to the rest of the code samples, and compared to the first ten lines. Should be 2 spaces per tab.